signing_service/signing_service/tests/daily_tresholds/2018-11-06

```diff
@@ -0,0 +1 @@
+0
```

Comment:
I guess this file appeared here accidentally?

Reply:
true, removing